vars/letsencrypt_events.yml

```diff
@@ -0,0 +1,8 @@
+# Walktrough: https://github.com/ZeusWPI/ansible-config/wiki/Steps-for-LetsEncrypt
+letsencrypt_webroot_path: /var/www/letsencrypt
+letsencrypt_email: admin@events.ugent.be
```

Comment:
Maybe change to admin-events@zeus.ugent.be? Because events.ugent.be doesn't have MX records.